src/routes/console/organization-[organization]/settings/deleteOrganizationModal.svelte

```diff
@@ -46,34 +59,128 @@
         }
     }
 
+    const tabs = [
+        { name: 'projects', label: 'Projects', total: $projects.total },
+        { name: 'members', label: 'Team members', total: $members.total }
+    ];
+
+    $: tabData =
+        selectedTab === 'projects'
+            ? {
+                  headers: ['Project', '', 'Last Updated'],
+                  rows: $projects.projects.map((project) => ({
+                      cells: [project.name, '', toLocaleDate(project.$updatedAt)]
+                  }))
+              }
+            : {
+                  headers: ['Member', '', 'Last Activity'],
+                  rows: $members.memberships.map((member) => ({
+                      cells: [member.userName, '', toLocaleDate(member.$updatedAt)]
+                  }))
+              };
+
     $: upcomingInvoice = invoices?.invoices.find((i) => i.status === 'upcoming' && i.amount > 0);
+
+    $: if (!showDelete) {
+        // reset on close.
+        organizationName = '';
+    }
 </script>
 
-<Modal
-    title="Delete organization"
-    onSubmit={deleteOrg}
-    bind:show={showDelete}
-    bind:error
-    icon="exclamation"
-    state="warning"
-    headerDivider={false}>
-    {#if upcomingInvoice}
-        <p data-private>
-            The organization <b>{$organization.name}</b> will be flagged for deletion.
-        </p>
+<div class="max-height-dialog">
+    <Modal
+        title="Delete organization"
+        onSubmit={deleteOrg}
+        bind:show={showDelete}
+        bind:error
+        icon="exclamation"
+        state="warning"
+        headerDivider={false}>
+        {#if upcomingInvoice}
+            <Alert type="warning">
+                <span slot="title">
+                    You have a pending {formatCurrency(upcomingInvoice.grossAmount)} invoice for your
+                    {tierToPlan(upcomingInvoice.plan).name} plan
+                </span>
+                <p>
+                    By proceeding, your invoice will be processed within the hour. Upon successful
+                    payment, your organization will be deleted.
+                </p>
+            </Alert>
+        {/if}
 
         <p data-private>
-            All existing projects will be paused and the organization will be deleted once your
-            upcoming invoice is processed on {toLocaleDate($organization.billingNextInvoiceDate)}.
-        </p>
-    {:else}
-        <p data-private>
-            Are you sure you want to delete <b>{$organization.name}</b>? All projects ({$projects.total})
-            and data associated with this organization will be deleted. This action is irreversible.
+            {#if $projects.total > 0}
+                The following projects and all data associated with <b>{$organization.name}</b> will
+                be permanently deleted. <b>This action is irreversible</b>.
+            {:else}
+                <b>{$organization.name}</b> will be permanently deleted.
```

Comment:
I made a small change to the copy for the single member/no projects use case. Otherwise looks good!

```suggestion
                All data associated with <b>{$organization.name}</b> will be permanently deleted.
```